src/main/markdown/steering.md

```diff
@@ -9,13 +9,11 @@ The committee will lead the GWT project in major decisions by consensus and ensu
 
 The steering committee consists of the following members (no particular order):
 
-* Goktug Gokdogan, [Google](https://www.google.com/about/)
 * Justin Hickman, [Vertispan](https://vertispan.com/)
```

Comment:
Maybe the "steering committee" article can be removed and links to it replaced by a link to a list of maintainers, e.g. https://github.com/orgs/gwtproject/people . That's not in the scope of this PR though. @FrankHossfeld @niloc132 @jhickman @tbroyer @vegegoku looped